duckchat/duckchat-impl/src/main/java/com/duckduckgo/duckchat/impl/inputscreen/ui/InputScreenConfigResolver.kt

```diff
@@ -55,9 +55,9 @@ class InputScreenConfigResolverImpl @Inject constructor(
         _isTopOmnibar = params?.isTopOmnibar ?: true
     }
 
-    override fun useTopBar(): Boolean =
+    override fun useTopBar(isLandscape: Boolean): Boolean =
         useTopBar(
-            isTopOmnibar = isTopOmnibar,
+            isTopOmnibar = isTopOmnibar || isLandscape,
             duckChatInternal = duckChatInternal,
         )
```

Comment:
```suggestion
    override fun useTopBar(isLandscape: Boolean): Boolean =
useTopBar(
            isTopOmnibar = isTopOmnibar,
duckChatInternal = duckChatInternal,
) || isLandscape
```

This is a little clearer in intent to me.